test/helpers/eip712-types.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  EIP712Domain: [
+    { name: 'name', type: 'string' },
+    { name: 'version', type: 'string' },
+    { name: 'chainId', type: 'uint256' },
+    { name: 'verifyingContract', type: 'address' },
+    { name: 'salt', type: 'bytes32' },
+  ],
+  Permit: [
+    { name: 'owner', type: 'address' },
+    { name: 'spender', type: 'address' },
+    { name: 'value', type: 'uint256' },
+    { name: 'nonce', type: 'uint256' },
+    { name: 'deadline', type: 'uint256' },
+  ],
+};
```

Comment:
I'm pretty sure we're missing a bunch of these. At least `ForwardRequest` from meta transactions is coming to my mind, I'll look for definitions and add to this file

Reply:
Moved them